src/Maths/Silk.NET.Maths/Scalar.Constants.cs

```diff
@@ -108,6 +108,8 @@ public static partial class Scalar<T> where T : notnull
         /// This is equivalent to <c>MathF.Pi / 180</c>.
         /// </remarks>
         public static readonly T RadiansPerDegree;
+        
+        internal static readonly T AllBitsSet = Scalar.Not(Scalar<T>.Zero);
```

Comment:
Fun fact: it works, but it doesn't have to work, since `Zero` is not initialized at this moment
(TODO to fix it)